mysql-test/suite/rpl/t/rpl_show_slave_hosts.test

```diff
@@ -375,31 +393,30 @@ let $status_var= Rpl_semi_sync_master_status;
 let $status_var_value= ON; 
 source include/wait_for_status_var.inc;
 show status like 'Rpl_semi_sync_master_status';
+show variables like 'rpl_semi_sync_master_enabled';
 
 # Write the new event
 INSERT INTO t VALUES (10);
 
-let $status_var= Rpl_semi_sync_master_status;
```

Comment:
We shouldn't delete the expected behavior out. I suppose we can adapt the test to allow it so it passes (and the actual `gtid_state_sent`/`gtid_state_ack` is consistent); but we should file a new MDEV about the broken behavior, comment out (but leave) code that supports the expected behavior, and add a comment referencing the new MDEV to un-comment the "good" path with the fix. We should follow this pattern for all the current broken semi-sync behaviors too, e.g. that described in test case 6:

```
# It may happen that rpl_semi_sync_master_no_transactions increments for above
# query.
```

Reply:
Agree have added MDEV-33615